### PR TITLE
Avoid using "Reactive" for Quarkus REST in security doc

### PR DESCRIPTION
**Pull request by gsmet**
Also remove a redundant section.

### CODE REVIEW

File: docs/src/main/asciidoc/security-openid-connect-client-reference.adoc [hunk 1]
```diff
@@ -1369,24 +1369,6 @@ As mentioned, use `AccessTokenRequestFilter` if you work with Keycloak or an Ope
 You can generate the tokens as described in xref:security-oidc-bearer-token-authentication.adoc#bearer-token-integration-testing[OpenID Connect Bearer Token Integration testing] section.
 Prepare the REST test endpoints. You can have the test front-end endpoint, which uses the injected MP REST client with a registered token propagation filter, call the downstream endpoint. For example, see the `integration-tests/resteasy-client-oidc-token-propagation` in the `main` Quarkus repository.
 
-[[reactive-token-propagation]]
-== Token Propagation Reactive
```
Comment (gsmet) [inline docs/src/main/asciidoc/security-openid-connect-client-reference.adoc:1373]: From what I can see, this section is redundant with the one above ` Token Propagation for Quarkus REST`.

File: docs/src/main/asciidoc/security-openid-connect-client-reference.adoc [hunk 2]
```diff
@@ -1369,24 +1369,6 @@ As mentioned, use `AccessTokenRequestFilter` if you work with Keycloak or an Ope
 You can generate the tokens as described in xref:security-oidc-bearer-token-authentication.adoc#bearer-token-integration-testing[OpenID Connect Bearer Token Integration testing] section.
 Prepare the REST test endpoints. You can have the test front-end endpoint, which uses the injected MP REST client with a registered token propagation filter, call the downstream endpoint. For example, see the `integration-tests/resteasy-client-oidc-token-propagation` in the `main` Quarkus repository.
```
Comment (gsmet) [inline docs/src/main/asciidoc/security-openid-connect-client-reference.adoc:1370]: Note for this PR but please let's not point to random content in our repo without even giving a link.

Please include some code here as an example.

Comment (sberyozkin) [reply docs/src/main/asciidoc/security-openid-connect-client-reference.adoc:1370]: Hi @gsmet Sure, I must've typed it with the very initial version of this doc, agree it is a bad idea.
How about linking to the corresponding quickstart section, https://quarkus.io/guides/security-openid-connect-client#testing-the-application ? There is a lot of code involved, and the quickstart covers it all for OIDC client and propagation.
If you agree, I can push a commit to your PR adding it.

Comment (sberyozkin) [reply docs/src/main/asciidoc/security-openid-connect-client-reference.adoc:1370]: So we replace `integration-tests/resteasy-client-oidc-token-propagation` with a link. Let me know what you prefer

**Review by sberyozkin — APPROVED** [review]

File: docs/src/main/asciidoc/security-openid-connect-client-reference.adoc (hunk 2 above)
Comment (sberyozkin) [reply docs/src/main/asciidoc/security-openid-connect-client-reference.adoc:1370]: Or I can follow up a bit later with  dedicated PR, whatever you prefer works for me

Comment (gsmet) [reply docs/src/main/asciidoc/security-openid-connect-client-reference.adoc:1370]: Yes, please do a follow-up!

**Pull request merged**